dist/pptxgen.cjs.js

```diff
@@ -4444,7 +4444,7 @@ function makeChartType(chartType, data, opts, valAxisId, catAxisId, isMultiTypeC
             strXml += '<c:' + chartType + 'Chart>';
             strXml += '  <c:varyColors val="1"/>';
             strXml += '<c:ser>';
-            if (opts.chartPieExplosion) {
+            if (opts.chartPieExplosion && Number.isInteger(opts.chartPieExplosion)) {

```

Comment:
This change belongs to previous commit; the bundled output was not added to the commit